deploy/flux-deployment.yaml

```diff
@@ -93,7 +93,10 @@ spec:
         # or with a different service name, you can supply these
         # following two arguments to tell fluxd how to connect to it.
         # - --memcached-hostname=memcached.default.svc.cluster.local
-        # - --memcached-service=memcached
+
+        # use the memcached ClusterIP service name by setting the
+        # memcached-service to string empty
+        - --memcached-service=
```

Comment:
Should the example service have the `clusterIP: None` line removed, in this case?